roxie/udplib/udptrr.cpp

```diff
@@ -574,6 +577,7 @@ class CReceiveManager : implements IReceiveManager, public CInterface
     {
         CReceiveManager &parent;
         ISocket *receive_socket;
```

Comment:
These should both be initialized to null, otherwise if the exception is throw on on line 590, the destructor will try to access uninitialised memory.